location/test.py

```diff
@@ -0,0 +1,11 @@
+import sys
+sys.path.append("../")
+from utils import db
+
+from location import get_location
+
+if __name__ == "__main__":
```

Comment:
we should probably write real unit tests with `unittest` or `pytest`. Even using doctest would be better than nothing.